salt/log/setup.py

```diff
@@ -905,6 +952,63 @@ def setup_multiprocessing_logging_listener(opts, queue=None):
     __MP_LOGGING_LISTENER_CONFIGURED = True
 
 
+def setup_multiprocessing_zmq_logging(port=None):
+    from salt.utils.platform import is_windows
+
+    global __MP_LOGGING_CONFIGURED
+    global __MP_LOGGING_ZMQ_HANDLER
+
+    if __MP_IN_MAINPROCESS is True and not is_windows():
+        # We're in the MainProcess, return! No multiprocessing logging setup shall happen
+        # Windows is the exception where we want to set up multiprocessing
+        # logging in the MainProcess.
+        return
+
+    try:
+        logging._acquireLock()  # pylint: disable=protected-access
+
+        if __MP_LOGGING_CONFIGURED is True:
+            return
+
+        # Let's set it to true as fast as possible
+        __MP_LOGGING_CONFIGURED = True
+
+        if os.getpid() in zmq_handlers:
+        #if __MP_LOGGING_ZMQ_HANDLER is not None:
```

Comment:
Debugging leftover?
The same for `zmq_handlers`

Reply:
yep, fixed